src/main/java/com/ghilly/exception/WrongArgumentNameException.java

```diff
@@ -0,0 +1,8 @@
+package com.ghilly.exception;
+
+public class WrongArgumentNameException extends RuntimeException {
```

Comment:
it's a bad name, because we don't know it's an argument or not

Reply:
Done